tests/udf/test_blobfinder.py

```diff
@@ -322,6 +324,57 @@ def test_run_refine_fullframe(lt_ctx):
     assert np.allclose(res['b'].data[0], b, atol=0.2)
 
 
+@pytest.mark.with_numba
+def test_run_refine_blocktests(lt_ctx):
+    shape = np.array([128, 128])
+    zero = shape / 2
+    a = np.array([27.17, 0.])
+    b = np.array([0., 29.19])
+    indices = np.mgrid[-2:3, -2:3]
+    indices = np.concatenate(indices.T)
+
+    radius = 7
+    match_pattern = blobfinder.RadialGradient(radius=radius)
+    crop_size = match_pattern.get_crop_size()
+
+    data, indices, peaks = cbed_frame(
+        *shape, zero=zero, a=a, b=b, indices=indices, radius=radius, margin=crop_size
+    )
+
+    dataset = MemoryDataSet(data=data, tileshape=(1, *shape),
+                            num_partitions=1, sig_dims=2)
+
+    template = match_pattern.get_template((2*crop_size, 2*crop_size))
+    nbytes = template.nbytes
+
+    udf_cs = (
+        blobfinder.FastCorrelationUDF,
+        blobfinder.FullFrameCorrelationUDF,
+    )
+
+    for udf_c in udf_cs:
+        for limit in (
```

Comment:
Generally, I would prefer to use pytest's parametrization instead of for-loops, because it allows to test the parameters independently and not abort after the first failure. That may give you a better picture what works and what is actually failing in the test case.

It may be difficult in this case, as the limit depends on local variables, but at least `udf_cs` could be [parametrize'd](https://docs.pytest.org/en/latest/parametrize.html).

Other than this, the PR looks good and ready to merge!

Reply:
Ok, done! :-)